nikola/image_processing.py

```diff
@@ -83,13 +83,15 @@ def filter_exif(self, exif, whitelist):
 
     def resize_image(self, src, dst, max_size, bigger_panoramas=True, preserve_exif_data=False, exif_whitelist={}, preserve_icc_profiles=False):
         """Make a copy of the image in the requested size."""
-        if os.path.splitext(src)[1] in ['.svg', '.svgz']:
+        extension = os.path.splitext(src)[1].lower()
+        if extension in ['.svg', '.svgz']:
```

Comment:
(We could use a {set} instead of a [list] for a negligible performance improvement.)

Reply:
Indeed, pushing it since it's easy